pkg/v1/tkg/web/src/app/views/landing/vsphere-wizard/node-setting-step/node-setting-step.component.ts

```diff
@@ -13,12 +13,18 @@ import { VSphereWizardFormService } from 'src/app/shared/service/vsphere-wizard-
  * App imports
  */
 import { IpFamilyEnum, PROVIDERS, Providers } from '../../../../shared/constants/app.constants';
-import { NodeType, vSphereNodeTypes } from '../../wizard/shared/constants/wizard.constants';
+import { NodeType} from '../../wizard/shared/constants/wizard.constants';
 import { StepFormDirective } from '../../wizard/shared/step-form/step-form';
 import { ValidationService } from '../../wizard/shared/validation/validation.service';
 import { KUBE_VIP, NSX_ADVANCED_LOAD_BALANCER } from '../../wizard/shared/components/steps/load-balancer/load-balancer-step.component';
 import Broker from 'src/app/shared/service/broker';
 import { AppEdition } from 'src/app/shared/constants/branding.constants';
+import { VsphereField, vSphereNodeTypes } from "../vsphere-wizard.constants";
+
+enum InstanceType {
```

Comment:
This looks like something that we can maybe move out to wizard.constants, but we can do it in a later PR if you prefer